### PR TITLE
fix ValueError #8

### PR DESCRIPTION
**Pull request by tatsu-i**
I solved the problem by providing multiple formats of strptime and verifying them

### CODE REVIEW

**Comment by mshudrak** [comment]
Hi [tatsu-i](https://github.com/tatsu-i)

Sorry for the long response, you need to sign CLA before I can take a look at your pull request. Thank you.

```
Thanks for your pull request! It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

View this [failed invocation](https://github.com/google/gcp_scanner/pull/9/checks?check_run_id=9155656357) of the CLA check for more information.

For the most up to date status, view the checks section at the bottom of the pull request.
```

**Comment by mshudrak** [comment]
Basically:
 If you are not currently covered under a CLA, please visit https://cla.developers.google.com/. Once you've signed, follow the "New Contributors" link at the bottom of this page to update this check.

File: src/gcp_scanner/credsdb.py [hunk 1]
```diff
@@ -224,8 +224,11 @@ def get_access_tokens_dict(path_to_creds_db: str) -> Dict[str, str]:
       token = row[1]
       expiration_date = row[2]
 
-      token_time_obj = datetime.datetime.strptime(expiration_date,
-                                                  "%Y-%m-%d %H:%M:%S.%f")
+      for fmt in ["%Y-%m-%d %H:%M:%S.%f", "%Y-%m-%d %H:%M:%S"]:
```
Comment (mshudrak) [inline src/gcp_scanner/credsdb.py:227]: I'd just omit .%f to avoid any errors and exceptions at all and make code as simple as possible.

Comment (tatsu-i) [reply src/gcp_scanner/credsdb.py]: I omitted the .%f to avoid the error

**Comment by tatsu-i** [comment]
@googlebot @google-cla I signed it!

**Pull request merged**